includes/config/default.yml

```diff
@@ -28,7 +30,7 @@ default:
     # Tests that are meant for development sites and can have config changes.
     dev:
       filters:
-        tags: '~contrib'
+        tags: 'dev,~contrib'
```

Comment:
I think I may want to not explicitly say run only dev tests but I also may only want to. Flip flopping on this one.